app/flows/check_travel_during_coronavirus_flow/outcomes/_travel_result_layout.erb

```diff
@@ -7,21 +7,23 @@
 
 <div class="govuk-grid-row travel-abroad__section">
   <div class="govuk-grid-column-one-third travel-abroad__reasons">
-    <p class="govuk-body govuk-body-s">Because you said</p>
-    <ul class="govuk-list govuk-list--bullet govuk-body-s">
-      <% data[:reasons].each do |reason| %>
-        <li><%= reason %></li>
-      <% end %>
-      <% if calculator.travelling_with_young_people? %>
-        <li>You are travelling with someone aged 17 and under</li>
-      <% end %>
-      <% if calculator.travelling_with_children.include?("zero_to_four") %>
-        <li>You are travelling with someone aged 4 and under</li>
-      <% end %>
-      <% if calculator.travelling_with_children.include?("five_to_seventeen") %>
-        <li>You are travelling with someone aged 5 to 17</li>
-      <% end %>
-    </ul>
+    <% unless calculator.travelling_to_ireland? && calculator.single_journey? %>
```

Comment:
I've moved these changes, I'll amend further because there's more comments on the next commit. Excellent shout with the `summary_text_fields.any?` thanks!